### DISCUSSION TITLE
류마티즘 환자를 위한 기록 앱!

### DISCUSSION THREAD

**Discussion by acciohyun** [DeveloperAcademy-POSTECH/.github#53] [Show and tell]
안녕하세요~ 3기 미아입니다 🤩.

류마티즘은 평생 함께하는 친구인데요. 병원을 2-3개월에 한번씩 가기에 환자가 본인의 컨디션을 잘 기록하며 관리를 해야 삶의 질이 확연히 달라질 수 있습니다. 

기록하고 관리하는 삶을 위해서 만들었습니다! 

App store에서 "Rapha"를 확인해보세요 ;)

https://apps.apple.com/kr/app/rapha-for-as/id6499498701?l=en-GB

꾸준히 유지보수 + 업데이트 하고 있습니다.

**Comment by yeniful** [comment]
안녕하세요! 글 서두에 몇 기의 누구인지도 함께 적어주시면 좋을 것 같아요 !
앞으로도 Rapha가 디벨롭이 되어서 류마티즘 환자분들께 도움이 더더욱 될 수 있기를 바랍니다-! 🙌
